documentation/objects/objects.md

```diff
@@ -69,7 +60,7 @@ In this applet this continuous range is transferred into a binary one, containin
 
 **Note:** To see the results of changing the parameter settings in this applet, press the "Apply" button.
 
-There are two algorithms you can choose from to threshold your data: _Simple_ and _Hysteresis_, which can be selected using the "Method" drop down.
+There are two algorithms you can choose from to threshold your data: _Simple_ and _Hysteresis_, which can be selected using the "Method" drop down. The most important difference between the two is that hysteresis thresholding makes it possible to separate adjacent objects.
```

Comment:
maybe "touching objects"? Things can be adjacent without touching...

Reply:
"touching objects" was my first version, but I found it much more vague than adjacent. I'll go with "connected" because in the end it's about connected components...